modules/plugins/utility/nvim-biscuits/nvim-biscuits.nix

```
@@ -0,0 +1,12 @@
+{lib, ...}: let
+  inherit (lib.options) mkEnableOption;
+  inherit (lib.nvim.types) mkPluginSetupOption;
+in {
+  options.vim.utility.nvim-biscuits = {
+    enable = mkEnableOption ''
+      A neovim port of Assorted Biscuits.
+    '';
```

```suggestion
    enable = mkEnableOption "a Neovim port of Assorted Biscuits [nvim-biscuits]";
```

Relatively minor change for consistency in documentation. For reference, `mkEnableOption` prepends "Whether to enable" to the string it give you so it would look like:

```
Whether to enable A neovim port...
```

Notice the inconsistent capital A.

---

Nice catch, I'll fix that quick. Thanks for the help!